src/main/java/com/sumologic/http/sender/SumoHttpSender.java

```diff
@@ -157,6 +165,13 @@ private void keepTrying(String body) {
                     break;
                 }
             }
+
+            if ((tries - 1 == maxNumberOfRetries) && (maxNumberOfRetries >= 0)) {
+                if (!success) {
+                    logger.warn("Dropping message, because max number of retires has been reached. Message: %s", body);
```

Comment:
```suggestion
                    logger.warn("Dropping message, because max number of retries has been reached. Message: %s", body);
```
```suggestion
                    logger.warn("Dropping message, because max number of retires has been reached. Message: %s", body);
```